Entities/Industry/Drill/Drill.as

```diff
@@ -190,7 +190,7 @@ void onTick(CBlob@ this)
 		this.Sync(heat_prop, true);
 	}
 	sprite.SetEmitSoundPaused(true);
-	if (this.isAttached())
+	if (this.isAttachedToPoint("PICKUP"))
```

Comment:
This does not appear to be necessary because of the next few lines:

```angelscript
AttachmentPoint@ point = this.getAttachments().getAttachmentPointByName("PICKUP");
CBlob@ holder = point.getOccupied();

if (holder is null || holder.isAttached()) return;
```